.taskcluster.yml

```diff
@@ -1,889 +1,77 @@
-  workerType: wpt-docker-worker
-- extra:
-    github:
-      branches: [master]
-      events: [push]
-  metadata: {description: '', name: wpt-firefox-nightly-testharness-14, owner: '{{
-      event.head.user.email }}', source: '{{ event.head.repo.url }}'}
-  payload:
-    artifacts:
-      public/results: {path: /home/test/artifacts, type: directory}
-    command: [/bin/bash, --login, -c, ">-\n            ~/start.sh {{event.head.repo.url}}\
-        \ {{ event.head.repo.branch }} {{event.head.sha}} firefox-nightly &&\n   \
-        \         cd ~/web-platform-tests &&\n            ./tools/ci/ci_taskcluster.sh\
-        \ firefox testharness 14 15"]
-    image: gsnedders/web-platform-tests:0.13
-    maxRunTime: 7200
-  provisionerId: aws-provisioner-v1
-  workerType: wpt-docker-worker
-- extra:
-    github:
-      branches: [master]
-      events: [push]
-  metadata: {description: '', name: wpt-firefox-nightly-testharness-15, owner: '{{
-      event.head.user.email }}', source: '{{ event.head.repo.url }}'}
-  payload:
-    artifacts:
-      public/results: {path: /home/test/artifacts, type: directory}
-    command: [/bin/bash, --login, -c, ">-\n            ~/start.sh {{event.head.repo.url}}\
-        \ {{ event.head.repo.branch }} {{event.head.sha}} firefox-nightly &&\n   \
-        \         cd ~/web-platform-tests &&\n            ./tools/ci/ci_taskcluster.sh\
-        \ firefox testharness 15 15"]
-    image: gsnedders/web-platform-tests:0.13
-    maxRunTime: 7200
-  provisionerId: aws-provisioner-v1
-  workerType: wpt-docker-worker
-version: 0
+  $if: tasks_for == "github-push"
+  then:
+    $if: event.ref == "refs/heads/master"
+    then:
+      $flattenDeep:
+        $map: [{name: firefox, channel: nightly}, {name: chrome, channel: dev}]
+        each(browser):
+          $map:
+            - [testharness, 1, 15]
+            - [testharness, 2, 15]
+            - [testharness, 3, 15]
+            - [testharness, 4, 15]
+            - [testharness, 5, 15]
+            - [testharness, 6, 15]
+            - [testharness, 7, 15]
+            - [testharness, 8, 15]
+            - [testharness, 9, 15]
+            - [testharness, 10, 15]
+            - [testharness, 11, 15]
+            - [testharness, 12, 15]
+            - [testharness, 13, 15]
+            - [testharness, 14, 15]
+            - [testharness, 15, 15]
+            - [reftest, 1, 10]
+            - [reftest, 2, 10]
+            - [reftest, 3, 10]
+            - [reftest, 4, 10]
+            - [reftest, 5, 10]
+            - [reftest, 6, 10]
+            - [reftest, 7, 10]
+            - [reftest, 8, 10]
+            - [reftest, 9, 10]
+            - [reftest, 10, 10]
+            - [wdspec, 1, 1]
+          each(chunk):
+            taskId: {$eval: 'as_slugid(browser.name + browser.channel + chunk[0] + str(chunk[1]))'}
+            taskGroupId: {$eval: 'as_slugid("task group")'}
+            created: {$fromNow: ''}
+            deadline: {$fromNow: '5 hours'}
```

Comment:
I think in gecko this is set to 1 day, but I can't open  taskcluster right now to check. That seems more sensible to me (c.f. the case today of @lukebjerring making a lot of IDL changes at once that caused a backlog).